Plugins/Gradle/build.gradle

```diff
@@ -22,11 +26,18 @@ ext {
     ]
 }
 
-version = "$project.versions.testify"
+version = gradleProperties.getProperty('testify_version')
 group = pom.publishedGroupId
-archivesBaseName = pom.artifact
 
-sourceCompatibility = JavaVersion.VERSION_17
+gradlePlugin {
+    automatedPublishing = false
```

Comment:
Probably a sign that we should move to letting the plugin generate the publications itself, but glad this is all it took.

Reply:
Absolutely not. Never touching publishing again unless something is deprecated or broken 😆 
I spent an embarrassing amount of time trying to get the built-in publishing to produce a properly signed plugin that was compatible with Maven Central. `gradlePlugin` is designed to publish to `https://plugins.gradle.org/` and the Maven Central support is confusing and seemingly impossible to configure properly.